### ISSUE TITLE
Error adding Reelflix

### ISSUE THREAD

**Issue by crafty35a** [Prowlarr/Prowlarr#956]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Attempting to add Reelflix (with an API token) results in an error: "Unable to connect to indexer, check the log for more details"

### Expected Behavior

Successfully add tracker.

### Environment

```markdown
- OS: UnRAID
- Prowlarr: 0.3.0.1687
- Docker Install: Yes
- Using Reverse Proxy: No
```

### What branch are you running?

Nightly

### Trace Logs?

[trace.txt](https://github.com/Prowlarr/Prowlarr/files/8491895/trace.txt)

[AB#2994](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/2994)

**Comment by bakerboy448** [comment]
Check your indexer definition; does it match - https://github.com/Prowlarr/Indexers/blob/master/definitions/v4/reelflix-api.yml ?

https://wiki.servarr.com/prowlarr/appdata-directory  `/definitions` subfolder

**Comment by crafty35a** [comment]
Yes, they look identical

**Comment by Avenred** [comment]
I'm having the same issue. My indexer definitions match. I'm running Prowlarr on Ubuntu 20.04 with Prowlarr version 0.2.0.1678

**Comment by Qstick** [comment]
Fixed in https://github.com/Prowlarr/Indexers/commit/b85cbc72d97b856ffba3249b515ebfd9f96bdc0c ?

**Comment by Avenred** [comment]
No. My .yml file matches that commit and it's not working.

**Comment by crafty35a** [comment]
> Fixed in [Prowlarr/Indexers@b85cbc7](https://github.com/Prowlarr/Indexers/commit/b85cbc72d97b856ffba3249b515ebfd9f96bdc0c) ?

Yes, it seems to be working now for me. Thanks!
